### PR TITLE
Update macos instructions

### PR DESCRIPTION
**Pull request by pravynandas**
Added note for MacOs /var/run/docke.sock path specification.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -152,6 +152,8 @@ Only those (if you access the Mastercontainer Interface internally via port 8080
 ### How to run AIO on macOS?
 On macOS, there is only one thing different in comparison to Linux: instead of using `--volume /var/run/docker.sock:/var/run/docker.sock:ro`, you need to use `--volume /var/run/docker.sock.raw:/var/run/docker.sock:ro` to run it after you installed [Docker Desktop](https://www.docker.com/products/docker-desktop/) (and don't forget to [enable ipv6](https://github.com/nextcloud/all-in-one/blob/main/docker-ipv6-support.md) if you should need that). Apart from that it should work and behave the same like on Linux.
 
+Update: On MacOS Ventura 13.5.1 (22G90) with Docker Desktop 4.22.1 (118664), the path is /var/run/docker.sock (same as linux).
```
Comment (szaimen) [inline readme.md:155]: If that is true we can probably simply change the text above?

**Comment by pravynandas** [comment]
It might still help old macOS users.

**Comment by szaimen** [comment]
> It might still help old macOS users.

I doubt this is coupled to the macOS version. Probably rather a Docker Desktop version improved this so it should be the same for all people that want to try AIO on macOS. So can you please adjust the documentation above? :)

**Comment by szaimen** [comment]
I guess I can close this?

**Pull request closed without merging**